src/storage/datacore/restapi/mode/statusmonitor.pm

```diff
@@ -0,0 +1,151 @@
+#
```

Comment:
As for alerts I guess status.pm could be a mode name more generic in comparison with other modes already develop in this project.